src/viewer-selector.js

```diff
@@ -124,6 +125,7 @@ ViewerSelector.prototype.createDialog_ = function(options) {
   s.background = '#fafafa';
   s.fontFamily = "'Roboto', sans-serif";
   s.boxShadow = '0px 5px 20px #666';
+  s["z-index"] = 120;
```

Comment:
Yes, it works when it's written that way as well as when it's written `s.zIndex`.